dotnet/dynamodb/UpdateItemDataModel/UpdateItemDataModel.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[dynamodb.dotnet35.UpdateItemDataModel]
+using System;
+// using System.Collections.Generic;
```

Comment:
If these aren't being used, why not just delete them?

Reply:
I have now.